tests/models/decoder_only/vision_language/test_idefics3.py

```diff
@@ -0,0 +1,194 @@
+from typing import List, Optional, Tuple, Type
+
+import pytest
+from transformers import AutoModelForVision2Seq, AutoTokenizer
+
+from vllm.multimodal.utils import rescale_image_size
+from vllm.sequence import SampleLogprobs
+
+from ....conftest import IMAGE_ASSETS, HfRunner, PromptImageInput, VllmRunner
+from ...utils import check_logprobs_close
+
+HF_IMAGE_PROMPTS = IMAGE_ASSETS.prompts({
+    "stop_sign":
+    "<|begin_of_text|>User:<image>What's the content of the image?<end_of_utterance>\nAssistant:",  # noqa: E501
+    "cherry_blossom":
+    "<|begin_of_text|>User:<image>What is the season?<end_of_utterance>\nAssistant:",  # noqa: E501
+})
+HF_MULTIIMAGE_IMAGE_PROMPT = "<|begin_of_text|>User:<image><image>Describe these images.<end_of_utterance>\nAssistant:"  # noqa: E501
+
+models = ["HuggingFaceM4/Idefics3-8B-Llama3"]
+target_dtype = "half"
+
+
+def vllm_to_hf_output(vllm_output: Tuple[List[int], str,
+                                         Optional[SampleLogprobs]],
+                      model: str):
+    """Sanitize vllm output to be comparable with hf output."""
+    _, output_str, out_logprobs = vllm_output
+
+    output_str_without_image = output_str
+
+    hf_output_str = output_str_without_image + "<end_of_utterance><|end_of_text|>"  # noqa: E501
+
+    tokenizer = AutoTokenizer.from_pretrained(model)
+    hf_output_ids = tokenizer.encode(output_str_without_image)
+    assert hf_output_ids[0] == 128000
+    hf_output_ids = hf_output_ids[1:]
+
+    return hf_output_ids, hf_output_str, out_logprobs
+
+
+def run_test(
+    hf_runner: Type[HfRunner],
+    vllm_runner: Type[VllmRunner],
+    inputs: List[Tuple[List[str], PromptImageInput]],
+    model: str,
+    *,
+    dtype: str,
+    max_tokens: int,
+    num_logprobs: int,
+    mm_limit: int,
+    tensor_parallel_size: int,
+    distributed_executor_backend: Optional[str] = None,
+):
```

Comment:
Since the test doesn't have special case like patching or hacking, we can integrate it to `test_models.py`.